src/CoreWCF.ConfigurationManager/src/CoreWCF/Configuration/IssuedTokenParametersElement.cs

```diff
@@ -0,0 +1,328 @@
+            //    newElement.Copy(xmlElement);
+            //    this.AdditionalRequestParameters.Add(newElement);
+            //}
+
+            //TODO: Implement IssuedTokenParameters
+            //foreach (ClaimTypeElement c in source.ClaimTypeRequirements)
+            //{
+            //    this.ClaimTypeRequirements.Add(new ClaimTypeElement(c.ClaimType, c.IsOptional));
+            //}
+
+            this.KeySize = source.KeySize;
+            this.KeyType = source.KeyType;
+            this.TokenType = source.TokenType;
+            this.DefaultMessageSecurityVersion = source.DefaultMessageSecurityVersion;
+            this.UseStrTransform = source.UseStrTransform;
+
+            if (PropertyValueOrigin.Default != source.ElementInformation.Properties[ConfigurationStrings.Issuer].ValueOrigin)
+            {
+                this.Issuer.Copy(source.Issuer);
+            }
+            if (PropertyValueOrigin.Default != source.ElementInformation.Properties[ConfigurationStrings.IssuerMetadata].ValueOrigin)
+            {
+                this.IssuerMetadata.Copy(source.IssuerMetadata);
+            }
+        }
+
+        internal IssuedSecurityTokenParameters Create(bool createTemplateOnly, SecurityKeyType templateKeyType)
+        {
+            IssuedSecurityTokenParameters result = new IssuedSecurityTokenParameters();
+            if (!createTemplateOnly)
+            {
+                this.ApplyConfiguration(result);
+            }
+            else
+            {
+                //TODO: This is an internal property
+                //result.KeyType = templateKeyType;
+            }
+            return result;
+        }
+
+        internal void InitializeFrom(IssuedSecurityTokenParameters source, bool initializeNestedBindings)
+        {
+            if (null == source)
+                throw DiagnosticUtility.ExceptionUtility.ThrowHelperArgumentNull("source");
+
+            SetPropertyValueIfNotDefaultValue(ConfigurationStrings.KeyType, source.KeyType);
+            if (source.KeySize > 0)
+            {
+                SetPropertyValueIfNotDefaultValue(ConfigurationStrings.KeySize, source.KeySize);
+            }
+            SetPropertyValueIfNotDefaultValue(ConfigurationStrings.TokenType, source.TokenType);
+            SetPropertyValueIfNotDefaultValue(ConfigurationStrings.UseStrTransform, source.UseStrTransform);
+
+            if (source.IssuerAddress != null)
+                this.Issuer.InitializeFrom(source.IssuerAddress);
+
+            if (source.DefaultMessageSecurityVersion != null)
+                SetPropertyValueIfNotDefaultValue(ConfigurationStrings.DefaultMessageSecurityVersion, source.DefaultMessageSecurityVersion);
+
+            if (source.IssuerBinding != null && initializeNestedBindings)
+            {
+                this.Issuer.BindingConfiguration = this.Issuer.Address.ToString();
+
+                //TODO: Implement IssuedTokenParameters
+                //string bindingSectionName;
```

Comment:
Always throw is it isn't implemented and there's an optional check where the code won't be run if not being used in config.